CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+Neuro_Extras v20.9.29a2 (2020-09-29)
```

Comment:
Let's follow the suggested by @anayden approach at https://github.com/neuro-inc/neuro-extras/blob/master/RELEASE.md